go/kbnm/handler.go

```diff
@@ -73,7 +82,7 @@ func (h *handler) handleChat(req *Request) error {
 	}
 
 	var out bytes.Buffer
-	cmd := exec.Command(binPath, "chat", "send", "--private", req.To)
+	cmd := exec.Command(binPath, "chat", "send", "--private", cleanCmdArg(req.To))
```

Comment:
There's no shell involved in `exec.Command`, just a raw `execve` call or whatever, so there's no such thing as a special character. I can only see a couple corner case reasons to strip these characters:

1. We're worried that someone might replace `exec.Command` with something that does run shell code. (Though I don't know if the standard lib even exposes that.)
2. We're worried that the chat command itself might do bad things with the characters if it gets its hands on them. (Probably better solved somewhere other than here.)

If we want to keep doing this for either of those reasons, let's leave a comment explaining why we did what we did. Also it might be better to directly return an error than to strip characters and hope for best?